tests/src/dir.targets

```diff
@@ -50,6 +50,10 @@
     <_CLRTestNeedsProjectToRun Condition=" '$(_CLRTestNeedsToRun)' and '!$(_CLRTestBuildsExecutable)' ">true</_CLRTestNeedsProjectToRun>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <DebugType Condition=" '$(DebugType)'=='' and '$(RunningOnUnix)' == 'true' ">None</DebugType>
```

Comment:
Could you add a comment as to why this is necessary?  I believe we want to remove it once we get a fixed Roslyn, but the comment will be helpful to future code explorers in case we let that fall through the cracks.  Please also open an issue to track removing this to make that less likely.